lib/puppet/type/java_ks.rb

```diff
@@ -152,6 +152,13 @@ def self.title_patterns
         ]
       ],
       [
+        /^(.*):([a-z]:(\/|\\).*)$/i,
+        [
+            [ :name, identity ],
+            [ :target, identity ]
+        ]
```

Comment:
It does, I have ran the acceptance tests, it is case statement style first match wins, and the first states no colon, second for two colons and the third a single colon